manifest.outlook.xml

```diff
@@ -49,11 +49,11 @@
         <Host xsi:type="MailHost">
           <DesktopFormFactor>
             <FunctionFile resid="Commands.Url" />
-            <ExtensionPoint xsi:type="MessageComposeCommandSurface">
+            <ExtensionPoint xsi:type="MessageReadCommandSurface">
```

Comment:
Don't make this change. The function in the task pane writes to the email, so it has to be in compose mode.

```suggestion
            <ExtensionPoint xsi:type="MessageComposeCommandSurface">
```